go/src/socialapi/models/helpers.go

```diff
@@ -260,6 +260,7 @@ func CreateAccountInBothDbsWithNick(nick string) (*Account, error) {
 			Salt:           accHex,
 			Name:           nick,
 			Email:          accHex + "@koding.com",
+			SanitizedEmail: accHex + "@koding.com",
```

Comment:
sanitized email is generated from given email on the [registration time](https://github.com/koding/koding/blob/master/workers/social/lib/social/models/user/index.coffee#L1038) and required since there is [an index](https://github.com/koding/koding/blob/master/workers/social/lib/social/models/user/index.coffee#L69) since we usually dont run tests locally (when i did ) this was failing due to unique index issue. not directly related with this PR